internal/server/admin/certificate_service.go

```diff
@@ -235,7 +258,49 @@ func certificatesFromObjects(objects []model.Object) ([]*pb.Certificate, error)
 				return nil, err
 			}
 		}
+		// retrieve the snis currently using this certificate
+		if err := addSNIMetadata(ctx, cert, db); err != nil {
```

Comment:
Ideally, this would happen within the same transaction to adhere to ACID, but that's easier said than done right now. This isn't likely to be a problem for most, but figured to mention it as it's something we should try to iterate on.

Nothing needs to be done for this right now though.